hub/powertoys/dsc-configure.md

```diff
@@ -0,0 +1,482 @@
+---
+title: PowerToys DSC Configure module
+description: DSC configure module documentation for PowerToys
+ms.date: 04/03/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, DSC, Win]
+---
+
+# DSC configuration
+
+Since version 0.80, PowerToys installer is released on GitHub with `Microsoft.PowerToys.Configure` [DSC resource](https://learn.microsoft.com/en-us/powershell/dsc/overview?view=dsc-2.0) that allows you to configure PowerToys using a [Winget configuration file](../package-manager/configuration/create).
```

Comment:
```suggestion
Since version 0.80, PowerToys installer is released on GitHub with `Microsoft.PowerToys.Configure` [DSC resource](/powershell/dsc/overview) that allows you to configure PowerToys using a [Winget configuration file](/windows/package-manager/configuration/create).
```